dspace-server-webapp/src/main/java/org/dspace/app/rest/security/EPersonRestPermissionEvaluatorPlugin.java

```diff
@@ -37,6 +37,7 @@
 public class EPersonRestPermissionEvaluatorPlugin extends RestObjectPermissionEvaluatorPlugin {
 
     private static final Logger log = LoggerFactory.getLogger(EPersonRestPermissionEvaluatorPlugin.class);
+    public static final String OPERATION_PASSWORD_CHANGE = "/password";
```

Comment:
This variable/constant doesn't look to be needed here, as it's also in  `EPersonPasswordReplaceOperation.OPERATION_PASSWORD_CHANGE`.  In other words, I think this entire line can be removed